docs/sphinx/api/default_ops.rst

```diff
@@ -638,31 +638,67 @@ operations, each operating on 2 qubits.
 Photonic Operations on Qudits
 =============================
 
-These operations are valid only on the `photonics` target which does not support the quantum operations above.
+These operations are valid only on the `photonics` target which does not support
+the quantum operations above.
 
-:code:`plus`
+:code:`create`
```

Comment:
@bettinaheim / @1tnguyen : Thoughts on this gate naming? I think it is also a built-in `Operator` for upcoming feature.
I think it _means_ the same would like to confirm that there isn't an issue...